src/archive/traits.rs

```diff
@@ -1,31 +1,12 @@
 #[async_trait::async_trait]
 pub trait ArchiveService {
-    type EvmLogSelection;
-    type EthTransactSelection;
-    type ContractsEventSelection;
-    type GearMessageEnqueuedSelection;
-    type GearUserMessageSentSelection;
-    type EventSelection;
-    type CallSelection;
     type Batch;
+    type BatchOptions;
```

Comment:
What do you think about using this name 
```suggestion
    type BatchSelectors;
```

Reply:
BatchOptions has a few non-selector fields like: limit, from_block, to_block
so i'd like to use more general name

Reply:
Got it!